contentcuration/contentcuration/settings.py

```diff
@@ -56,8 +56,8 @@
 INSTALLED_APPS = (
     'contentcuration',
     'django.contrib.admin',
-    'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.auth',
```

Comment:
Can you add a comment explaining the problem and include the URL to where you found the fix?